guides/v2.1/cloud/cdn/configure-fastly.md

```diff
@@ -129,6 +138,7 @@ You cannot use the Fastly service in Integration environments.
 		git add -A; git commit -m "Install Fastly"; git push origin <branch name>
 
 4. Merge the branch code with the `master` Integration branch.
+
 5. [Deploy]({{ page.baseurl }}/cloud/live/stage-prod-live.html) the code to
    Staging and Production.
 
```

Comment:
The line below this has the phrase "log into" whereas other places say "log in to".